test/known_issues/test-http-path-contains-unicode.js

```diff
@@ -10,11 +10,8 @@ const http = require('http');
 
 const expected = '/café🐶';
 
-assert.strictEqual(
-  expected,
-  '/caf\u{e9}\u{1f436}',
-  'Sanity check that string literal produced the expected string'
-);
+//Sanity check for café🐶 is café🐶
```

Comment:
The comment is actually not even necessary. It will be printed in case of a error and the `expected` value is visible as well.